cave/analyzer.py

```diff
@@ -459,6 +459,35 @@ def parameter_importance(self, modus, incumbent, output, num_params=4,
         self.evaluators.append(self.pimp.evaluator)
         return self.pimp
 
+    def importance_table(self):
+        """Create a html-table over all evaluated parameter-importance-methods.
+        Parameters are sorted after their average importance."""
+        parameters = [p.name for p in self.scenario.cs.get_hyperparameters()]
+        index, values, columns = [], [], []
+        columns = [e.name for e in self.evaluators]
+        # Sort parameters after average importance
+        p_avg = {p : np.mean([e.evaluated_parameter_importance[p] for e in self.evaluators
```

Comment:
I just realized that ranking the parameters like this might be problematic as the values are not directly comparable.
Maybe we could support an argument "sort_by". If the user set this, the parameter are sorted according to the performance of the method specified in "sort_by". If that parameter is None, we sort by ranking.